modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -150,6 +150,26 @@ public void deleteByExternalReferenceCode(String externalReferenceCode)
 			_objectDefinition, null);
 	}
 
+	@Override
+	public void deleteCurrentObjectEntry(
```

Comment:
To be honest, the names generated by the REST Builder are crazy!!

`deleteCurrentObjectEntry` means delete the relationship between two objectEntries :clown_face: